docs/website/docs/getting-started/use-an-existing-source.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Use an existing source
+description: Use an existing source - how to leverage what others did before you
+keywords: [Use an existing a data source]
+---
+
+# How to use an existing verified source
+
+While dlt is primarily a pipeline building tool, on dlthub we host some common pipelines and also encourage the community to build and distribute high quality pipelines.
+Read on.
+
+## What is not a verified source but still widely available?
+
+We have a GPT-4 assistant on our docs, which will give you a code sample if you ask it nicely.
+Take note, it's meant to be a docs assistant first, so it might refuse your request if not
+worded as aligned with its goal.
+
+Or, join our [Slack community](https://join.slack.com/t/dlthub-community/shared_invite/zt-1slox199h-HAE7EQoXmstkP_bTqal65g) and ask a fellow human!
+
+## What is a verified source?
+
+A verified source is a source which we test daily in our verified sources repository.
```

Comment:
I'd explain more:
- We reviewed both the code and generated datasets and wrote demos and docs for them
- Many of them already in production.
- All of them are constantly tested on real data and distributed as simple Python code so they can be easily customized or hacked.